src/test/java/org/broadinstitute/hellbender/utils/IntervalUtilsUnitTest.java

```diff
@@ -1120,6 +1120,22 @@ public void testUnmergedIntervals(String unmergedIntervals) {
         Assert.assertEquals(merged.size(), 1);
     }
 
+    @Test
+    public void testIntervalFileIntervalsAreTrimmed() throws Exception {
+        final GenomeLoc expectedGenomeLoc = hg19GenomeLocParser.createGenomeLoc("1", 1, 10);
+        final File gatkIntervalFile = createTempFile("testIntervalFileIntervalsAreTrimmed", ".intervals.list",
```

Comment:
Huh, I've never seen this method which creates the temp file and writes to it before.   That seems useful.  Maybe it should be promote somewhere.

Reply:
I moved it to IOUtils with other methods with similar functionality, though now its no longer test-only.